docker/backend.dockerfile

```diff
@@ -127,6 +127,8 @@ RUN chmod +x /start.sh
 COPY ./docker/start-reload.sh /start-reload.sh
 RUN chmod +x /start-reload.sh
 
+ENV PYTHONPATH=/app
```

Comment:
Idem, to sync with last changes:

https://github.com/Cytomine-ULiege/pims/blob/859b175621372683c8ce3d72761d6bc37179cd05/docker/backend.dockerfile#L133